jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpFields.java

```diff
@@ -149,6 +150,43 @@ static Mutable build(HttpFields fields, EnumSet<HttpHeader> removeFields)
         return new org.eclipse.jetty.http.MutableHttpFields(fields, removeFields);
     }
 
+    static Mutable build(HttpCompliance httpCompliance, BiConsumer<ComplianceViolation, String> notifyViolation)
+    {
+        return new org.eclipse.jetty.http.MutableHttpFields()
```

Comment:
Wont this show up as an anonymous class everywhere?

Reply:
good point. I'll make it a named one.